tests/fixture/erlinit_fixture.c

```diff
@@ -488,3 +489,9 @@ REPLACE(int, ioctl, (int fd, unsigned long request, ...))
     log("ioctl(%s)", req);
     return 0;
 }
+
+REPLACE(int, prlimit, (pid_t pid, __rlimit_resource_t resource, const struct rlimit *new_limit, struct rlimit *old_limit))
+{
+    log("prlimit(%d, %d, new_limit, old_limit)", pid, resource);
```

Comment:
It feels like printing out new_limit is important part for testing this.  On `resource`, do you know if the constants are specified by POSIX or are they OS dependent? I suppose that I might find out when I try it on OSX...

Reply:
I'm not sure. I had assumed it was OS dependent but only really because i didn't think that much about it

Reply:
The resource numbering is OS-dependent. I just looked at the man page and there are a bunch of Linux-specific ones. `prlimit` is also Linux-specific according to the man page.